ichub-backend/controllers/fastapi/app.py

```diff
@@ -114,4 +114,36 @@ async def twin_management_share_catalog_part_twin(catalog_part_twin_share: Catal
 
 @app.post("/twin-management/twin-aspect", response_model=TwinAspectRead, tags=["Twin Management"])
 async def twin_management_create_twin_aspect(twin_aspect_create: TwinAspectCreate) -> TwinAspectRead:
-    return twin_management_service.create_twin_aspect(twin_aspect_create)
\ No newline at end of file
+    return twin_management_service.create_twin_aspect(twin_aspect_create)
+
+@app.get("/submodel-dispatcher/{semantic_id}/{global_id}", response_model=Dict[str, Any], tags=["Submodel Dispatcher"])
+async def submodel_dispatcher_get_submodel_content(semantic_id: str, global_id: UUID, request: Request) -> Dict[str, Any]:
+    return _submodel_dispatcher_get_submodel_content(semantic_id, global_id, request)
+    
+@app.get("/submodel-dispatcher/{semantic_id}/{global_id}/submodel", response_model=Dict[str, Any], tags=["Submodel Dispatcher"])
+async def submodel_dispatcher_get_submodel_content_submodel(semantic_id: str, global_id: UUID, request: Request) -> Dict[str, Any]:
+    return _submodel_dispatcher_get_submodel_content(semantic_id, global_id, request)
+
+@app.get("/submodel-dispatcher/{semantic_id}/{global_id}/submodel/$value", response_model=Dict[str, Any], tags=["Submodel Dispatcher"])
```

Comment:
```suggestion
@app.get("/submodel-dispatcher/{semantic_id}/{global_id}/submodel/", response_model=Dict[str, Any], tags=["Submodel Dispatcher"])
```
What is the purpose of `$value`? I mean, in what case would it be used?

Reply:
Hi Carlos. That's acutally needed - consumer are intended to add /$value to the href attribute of the DTR entry in order to retrieve the JSON. It was before documented somewhere in Industry Core KIT - but seems they moved it now. I found a respective hint here: https://eclipse-tractusx.github.io/docs-kits/kits/digital-twin-kit/software-development-view/interaction-patterns#1-fetching-a-suppliers-twin
Nevertheless - that's a pretty awkward procedure and from my experience I found that sometimes consumers don't add this. That's why I implemented it (for the moment) in all three ways (without /submodel, with /submodel and with /submodel/$value). That's maybe a point to discuss with Markus Keidl from BMW/Industry Core comittee.

Reply:
Thank you for the information. If it's documented, we can leave it as is. We can always revisit and discuss it later, but for now, let's keep it as a precaution.